### ISSUE TITLE
plot_walks() fails without truth values

### ISSUE THREAD

**Issue by wagoner47** [Samreay/ChainConsumer#66]
I'm trying to plot my walkers to check if my burn-in is long enough, and I noticed that this seems to be failing now if I don't have truth values (see traceback below for details). It looks like it is trying to plot the truth values even though they aren't there. If I instead set all of the truth values to something arbitrary, I no longer get the error message, but I do end up with lines that don't actually belong on my plot. I also noticed that in the code, there is no check to see if there is a truth value for a given parameter before plotting the truth, so that could potentially be the problem.

I'm not sure if this is related to issue #61 ?

```
TypeError                                 Traceback (most recent call last)
<ipython-input-469-51f5f98dc0a3> in <module>()
----> 1 c.plotter.plot_walks(truth=None)

~/anaconda3/lib/python3.6/site-packages/chainconsumer/plotter.py in plot_walks(self, parameters, truth, extents, display, filename, chains, convolve, figsize, plot_weights, plot_posterior, log_weight)
    397                         chain_row = chain.get_data(p)
    398                         self._plot_walk(ax, p, chain_row, extents=extents.get(p), convolve=convolve, color=chain.config["color"])
--> 399                 self._plot_walk_truth(ax, truth.get(p))
    400             else:
    401                 if i == 0 and plot_posterior:

~/anaconda3/lib/python3.6/site-packages/chainconsumer/plotter.py in _plot_walk_truth(self, ax, truth)
   1123 
   1124     def _plot_walk_truth(self, ax, truth):
-> 1125         ax.axhline(truth, **self.parent.config_truth)
   1126 
   1127     def _convert_to_stdev(self, sigma):  # pragma: no cover

~/anaconda3/lib/python3.6/site-packages/matplotlib/axes/_axes.py in axhline(self, y, xmin, xmax, **kwargs)
    720         self._process_unit_info(ydata=y, kwargs=kwargs)
    721         yy = self.convert_yunits(y)
--> 722         scaley = (yy < ymin) or (yy > ymax)
    723 
    724         trans = self.get_yaxis_transform(which='grid')

TypeError: '>' not supported between instances of 'float' and 'NoneType'
```

**Comment by Samreay** [comment]
Yeah, looks like a definite bug, Ill jump on this as soon as I can!

**Comment by Samreay** [comment]
Should now be fixed in v0.26.2
